.secureli.yaml

```diff
@@ -9,5 +9,8 @@ repo_files:
   - .png
   - .jpg
   max_file_size: 1000000
+scan_patterns:
+  custom_scan_patterns:
+  - foo
```

Comment:
I've removed this, looks like this is being added as part of the unit testing for the new custom regex scan. We'll probably want to clean that up so that the secureli yaml file reverts to its state pre-unit test, or update the unit test to use a mock instead of actually requiring an update to the config